src/components/display/List.tsx

```diff
@@ -66,18 +66,26 @@ const StyledDiv = styled.div<{
 		)};
 `;
 
+type ItemProps = {
+	id: string;
+	onClick: (evt: unknown) => void;
```

Comment:
Why unknown and not a generic Event?

Reply:
I agree, Event is a better fit

Reply:
Also, could the onClick prop be optional?